lib/src/deprecation.dart

```diff
@@ -166,6 +166,13 @@ enum Deprecation {
         'Global built-in functions that are available in sass: modules.',
   ),
 
+  /// Deprecation for functions named "type".
+  typeFunction(
+    'type-function',
+    deprecatedIn: '1.86.0',
+    description: 'Functions named "type".',
+  ),
+
```

Comment:
off-topic: for whatever reason when I generate the deprecations I end up getting a different formatting, like:

```dart
  typeFunction('type-function', deprecatedIn: '1.86.0',
    description: 'Functions named "type".'),
```

do you happen to know what that would be? this is when running `dart run grinder`... I guess I may have some config file somewhere that tells dart to behave this way 🤔

Reply:
I saw something similar and I couldn't figure out exactly what it was. The Dart formatter is in the process of rolling out new changes so it's probably something related to that, but I don't know what exactly. I suspect making the generator add a trailing comma would force it to behave the same way for both the old and new formatter logic, though.